projects/github-actions/required-review/CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.1.0] - 2023-05-11
+### Added
+- Added feature where if a review requirement is not met, then request the review.
+
+### Changed
+- Updates misspelling
```

Comment:
Good news, you do not need to fill this in. This changelog will be automatically updated when we next release a new version of this action, based off the different changelog entries in the `changelog/` directory (where you've added your own changelog entry already, thank you).

```suggestion
```